server/services/participants.js

```diff
@@ -71,6 +71,32 @@ const setParticipantStatus = async (
   return { status };
 });
 
+const getParticipantByID = async (participantInfo) => dbClient.db.withTransaction(async (tx) => {
+  const participant = await tx[collections.PARTICIPANTS].findDoc({
+    id: participantInfo.id,
+  });
+  return participant;
+});
```

Comment:
I think there’s no reason to use a transaction for a simple find operation only, you can just jump to the findDoc directly.